py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -316,31 +315,27 @@ def _request(self, method, url, body=None):
                 response = http.request(method, url, body=body, headers=headers)
             statuscode = response.status
         data = response.data.decode("UTF-8")
-        LOGGER.debug(f"Remote response: status={response.status} | data={data} | headers={response.headers}")
+        LOGGER.debug(f"Remote response: data={data} | headers={response.headers}")
         try:
             if 300 <= statuscode < 304:
                 return self._request("GET", response.headers.get("location", None))
             if 399 < statuscode <= 500:
-                return {"status": statuscode, "value": data}
+                return {"value": data}
             content_type = []
             if response.headers.get("Content-Type", None):
                 content_type = response.headers.get("Content-Type", None).split(";")
             if not any([x.startswith("image/png") for x in content_type]):
                 try:
                     data = utils.load_json(data.strip())
                 except ValueError:
-                    if 199 < statuscode < 300:
-                        status = ErrorCode.SUCCESS
-                    else:
-                        status = ErrorCode.UNKNOWN_ERROR
-                    return {"status": status, "value": data.strip()}
+                    return {"value": data.strip()}
```

Comment:
not sure how the upstream code would understand if this is an erroneous response or a successful one after the status field is removed

Reply:
Maybe they can be removed, but removed from this pr to reduce diff